cou/steps/plan.py

```diff
@@ -95,6 +95,23 @@ def add_message(cls, message: str, message_type: MessageType = MessageType.WARNI
                 cls.warning_messages.append(message)
 
 
+async def verify_cloud(analysis_result: Analysis, args: CLIargs) -> None:
+    """Verify the cloud is ready for upgrade.
+
```

Comment:
The docstring could explain what it means to verify here.  It's not obvious, because the function returns None and doesn't mention raising exceptions.

Reply:
It can be too verbose here to list down what we check

Reply:
I mean something along the lines of "run a sequence of verification functions which populate the PlanStatus singleton with messages when errors are found".